sources/osgViewer/Viewer.js

```diff
@@ -449,7 +449,9 @@ define( [
         beginFrame: function () {
             this._startFrameTick = Timer.instance().tick();
         },
-
+        setEndFrameCallBack: function ( u ) {
```

Comment:
why this accerssors ? is viewer.getCamera().setFinalDrawCallback( cb ) not enough ?